commands/auth.ts

```diff
@@ -227,6 +228,7 @@ exports.builder = yargs => {
     },
   });
 
+  await addVerboseDescribe(yargs, i18n(`${i18nKey}.verboseDescribe`));
```

Comment:
I don't love that we have to make the builders async for this to work, but I couldn't find a way to get around it. `yargs.showHelp()` is async and we need it to grab the original help output for the command.

Reply:
Fixed with the new `makeYargsBuilder` util